data/tooling-data.yaml

```diff
@@ -901,6 +901,8 @@
   supportedDialects:
     draft: [6, 7, 2019-09, 2020-12]
   toolingListingNotes: 'Powered by JsonSchema.Net in Blazor WASM for client-side validation'
+  landscape:
```

Comment:
Why we have a parent property called Landscape? I think the logo is an important property and we don't need to group them into "landscape". I suggest avoiding it and creating the logo as any other property.

Reply:
The data model is implemented from the json schema eco-system tooling self identification schema, since it already had a landscape property with a logo field, I utilised the exact same model as I understood that the logo is more relevant to the landscape project. If you would like to instead extend the self identification schema and add logo as a root property, let me know.

Reply:
For now, the landscape property only exists in the schema and type interface but I only added a single logo to test it out. So, we can still update our data model slights to move the logo field to the root properties instead.

Reply:
Ok